static/js/util/util.js

```diff
@@ -428,3 +428,14 @@ export function findCourseRun(
 export const classify: (s: string) => string = (
   R.compose(R.replace(/_/g,'-'), _.snakeCase, R.defaultTo(""))
 );
+
+/**
+ * Return first and last names if available else username
+ */
+export function getUserDisplayName(profile: Profile): string {
+  let first = profile.first_name || profile.username;
+  let last =  profile.last_name || '';
+  let preferred_name = profile.preferred_name ? `(${profile.preferred_name})` : '';
+
+  return `${first} ${last} ${preferred_name}`;
```

Comment:
nitpick: can you make this `${first} ${last}${preferred_name}` and move the space into the `preferred_name` value so we don't have an extra space?